Index: language/languages.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\nreturn [\r\n    'english' => [\r\n        'name'           => 'English',\r\n        'locale'         => 'en',\r\n        'html_direction' => 'ltr',\r\n        'date_format'    => 'Y-m-d H:i:s',\r\n    ],\r\n];
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- language/languages.php	(revision 0a2e845101913ac186e3f4fc4b2676363aeb0f15)
+++ language/languages.php	(date 1599209786916)
@@ -6,4 +6,11 @@
         'html_direction' => 'ltr',
         'date_format'    => 'Y-m-d H:i:s',
     ],
+
+    'latvian' => [
+        'name'           => 'Latviešu',
+        'locale'         => 'lv',
+        'html_direction' => 'ltr',
+        'date_format'    => 'd.m.Y H:i:s',
+    ],
 ];
\ No newline at end of file
